tests/test/test.nim

```diff
@@ -0,0 +1,10 @@
+discard """
```

Comment:
rename to tnimout? and maybe make the test more strict, eg also add runtime output and an output spec?